src/Parser.js

```diff
@@ -62,9 +58,11 @@ export function parse (value, points, declarations, rules, rulesets) {
 					// ;
 					case 59 + offset:
 						if (length > 0)
-							append(declaration(temporary, length), declarations)
+							append(declaration(temporary + ';', length), declarations)
 						break
-					// rule/at-rule
+					// @ ;
```

Comment:
I'm confused by the `@` on this line, 59 is `;` and I don't know how `@` relates to this branch of code

Reply:
It is `;` but ones that a preceded by an at-rule i.e `@import '...';` at-rules terminated with semi-colons.